packages/story-editor/src/components/panels/design/layer/layerPanel.js

```diff
@@ -17,37 +17,24 @@
 /**
  * External dependencies
  */
-import { __ } from '@googleforcreators/i18n';
-import { useMemo, memo } from '@googleforcreators/react';
+import { memo } from '@googleforcreators/react';
 import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { Panel, PanelTitle, PanelContent } from '../../panel';
+import { PanelContent } from '../../panel';
 import useInspector from '../../../inspector/useInspector';
-import { TAB_HEIGHT, TAB_VERTICAL_MARGIN } from '../../../tabview';
 import LayerList from './layerList';
 import useLayers from './useLayers';
 
+const HEAD_AREA_HEIGHT = 64;
```

Comment:
Is the `HEAD_AREA_HEIGHT` related to the WordPress admin menu? Or does it include the WP admin menu + the buttons? Or just buttons?

In case it includes the WP admin menu, the value should come from `useConfig`,  e.g.
```
const { styleConstants: { topOffset } = {} } = useConfig();
```

If it's the menu area with buttons, let's use the `HEADER_HEIGHT` in the general constants file instead.

Reply:
was able to ditch this because the shared popup is accounting for available height already.